lib/vernier.rb

```diff
@@ -56,6 +57,13 @@ def self.stop_profile
     result
   end
 
+  def self.cancel_profile
+    if @collector
+      @collector.stop
```

Comment:
Should this be:

```suggestion
      @collector.cancel
```

Reply:
Good catch thanks. Still not working though :/